python/cudf/cudf/tests/test_parquet.py

```diff
@@ -210,6 +210,35 @@ def test_parquet_reader_basic(parquet_file, columns, engine):
     assert_eq(expect, got, check_categorical=False)
 
 
+@pytest.mark.filterwarnings("ignore:Using CPU")
+@pytest.mark.parametrize("engine", ["cudf"])
+def test_parquet_reader_empty_pandas_dataframe(tmpdir, engine):
+    df = pd.DataFrame()
+
+    fname = tmpdir.join("test_pq_reader_empty_pandas_dataframe.parquet")
+
+    df.to_parquet(fname)
+    assert os.path.exists(fname)
+
+    expect = pd.read_parquet(fname)
+    got = cudf.read_parquet(fname, engine="cudf")
+
+    if len(expect) == 0:
+        expect = expect.reset_index(drop=True)
+        got = got.reset_index(drop=True)
+        if "col_category" in expect.columns:
+            expect["col_category"] = expect["col_category"].astype("category")
+
+    # PANDAS returns category objects whereas cuDF returns hashes
+    if engine == "cudf":
+        if "col_category" in expect.columns:
+            expect = expect.drop(columns=["col_category"])
+        if "col_category" in got.columns:
+            got = got.drop(columns=["col_category"])
```

Comment:
This test doesn't use `category` so this chunk of code should be able to be removed.